app/viewmodels/helpers/events/MovementEventHelper.scala

```diff
@@ -459,4 +471,235 @@ class MovementEventHelper @Inject()(
       )
     )
   }
+
+  def rorDetailsCard(event: MovementHistoryEvent)(implicit movement: GetMovementResponse, messages: Messages): Html = {
+    movement.reportOfReceipt match {
+      case Some(reportOfReceipt) =>
+        val headingTitle: String = if (movement.destinationType == DestinationType.Export) {
+          messages(s"movementHistoryEvent.${event.eventType}.rorDetails.h2.export")
+        } else {
+          messages(s"movementHistoryEvent.${event.eventType}.rorDetails.h2")
+        }
+
+        val statusKey: String = if (movement.destinationType == DestinationType.Export) {
+          s"movementHistoryEvent.${event.eventType}.rorDetails.status.export"
+        } else {
+          s"movementHistoryEvent.${event.eventType}.rorDetails.status"
+        }
+
+        buildOverviewPartial(
+          headingId = None,
+          headingTitle = Some(headingTitle),
+          cardTitleMessageKey = None,
+          cardFooterHtml = None,
+          summaryListRows = Seq(
+            Some(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorDetails.dateOfArrival", reportOfReceipt.dateOfArrival.formatDateForUIOutput())),
+            Some(summaryListRowBuilder(statusKey, messages(s"movementHistoryEvent.${event.eventType}.rorDetails.status.value.${reportOfReceipt.acceptMovement}"))),
+            Some(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorDetails.moreInformation", reportOfReceipt.otherInformation.getOrElse(s"movementHistoryEvent.${event.eventType}.rorDetails.moreInformation.notProvided"))),
+          )
+        )
+      case None => Html("")
+    }
+  }
+
+  def rorConsigneeCard(event: MovementHistoryEvent)(implicit movement: GetMovementResponse, messages: Messages): Html = {
```

Comment:
Couldn't the existing `Consignee` card be updated for this? I'd have thought this content would have been consistent everwhere?